src/_sass/layout.scss

```diff
@@ -1417,7 +1418,6 @@ img.no-shadow {
 .formItem {
   display: block;
   position: relative;
-  padding-bottom: 1.25 * $spacing-unit;
```

Comment:
This moved to be `.subnav` margin as it turns out it's what was providing the spacing and was no longer visible w/o JS so would squish things.

Despite its name `.formItem` is only used for the search element so this won't affect anything else 😅